areas/cloudarchitect/src/AzureMcp.CloudArchitect/Services/CloudArchitectService.cs

```diff
@@ -0,0 +1,409 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
There's a lot of text in this file.  I wonder if there's a better way to do this?  Maybe store in json and load it?